packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionScreen.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC, useEffect, useMemo } from 'react';
+import { useAvailableMT5Accounts, useMT5AccountsList } from '@deriv/api';
+import { Provider } from '@deriv/library';
+import { qtMerge, Text } from '@deriv/quill-design';
+import { THooks } from '../../../../types';
+import { useDynamicLeverageModalState } from '../../components/DynamicLeverageContext';
+import { JurisdictionCard } from './JurisdictionCard';
+import { JurisdictionTncSection } from './JurisdictionTncSection';
+
+type TJurisdictionScreenProps = {
+    isCheckBoxChecked: boolean;
+    selectedJurisdiction: THooks.AvailableMT5Accounts['shortcode'];
+    setIsCheckBoxChecked: React.Dispatch<React.SetStateAction<boolean>>;
+    setSelectedJurisdiction: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
Import `Dispatch` in react and just called it here. We are trying to make everything consistent by importing from `React` rather than using `React.`

Reply:
Updated.